modules/claims_api/spec/requests/v1/power_of_attorney_request_spec.rb

```diff
@@ -115,7 +115,7 @@
             end
 
             context 'when consumer is Veteran and missing EDIPI' do
-              it 'catches a raised 422' do
```

Comment:
If we change the description, shouldn't we change the response status as well, or vice versa? but then this test will fail...

Reply:
Oops, good catch. I originally had this raising a `500`, but decided instead to add the error capture of `MPI::Errors::ArgumentError` in the Application Controller, to try to preserve the functionality as closely as possible. So the spec should say `catches a raised 422`, I just changed the description back to that